src/components/DatePicker/_DatePicker.scss

```diff
@@ -5,14 +5,22 @@ fieldset {
 
   &.datePicker {
     @include fieldset-reset;
+    margin-bottom: var(--space-m);
```

Comment:
This margin is not necessary, as we are expecting consuming apps to use `Form` to layout form fields.

Reply:
That's true but I noticed that when I added the `HelperErrorText` component to the whole `DatePicker` component, that it removed the margin-bottom rules. That's why I added it here but I can remove.